crates/swc_plugin_runner/src/wasix_runtime.rs

```diff
@@ -1,7 +1,36 @@
 use std::{path::PathBuf, sync::Arc};
 
+use parking_lot::Mutex;
+use swc_common::sync::{Lazy, OnceCell};
+use wasmer::Store;
 use wasmer_wasix::Runtime;
 
+/// A shared instance to plugin runtime engine.
+/// ref: https://github.com/wasmerio/wasmer/issues/3793#issuecomment-1607117480
+static ENGINE: Lazy<OnceCell<Mutex<wasmer::Engine>>> = Lazy::new(|| {
```

Comment:
Why do we use `Lazy<OnceCell<T>>>`? Lazy contains `OnceCell` internally

Reply:
probably a simple mistake while copy some codes.